README.md

```diff
@@ -3,6 +3,8 @@
 
 The [Adafruit GFX](https://github.com/adafruit/Adafruit-GFX-Library) introduces graphics primitives (points, lines, circles, etc.). This library add support for the [Newhaven NHD-3.12-25664UCY2](http://www.newhavendisplay.com/nhd31225664ucy2-p-3537.html) 256x64x16 Yellow OLED Display.  The driver supports 4-wire SPI mode.
 
+!! Do not use a version of the Adadfruit GFX library newer then v1.5.3 !!
```

Comment:
Why?

Reply:
Now I know why. >1.5.3 doesn't compile due to renamed variables in the GFX library.